packages/mui-material/src/Autocomplete/Autocomplete.js

```diff
@@ -450,6 +451,19 @@ const Autocomplete = React.forwardRef(function Autocomplete(inProps, ref) {
     groupedOptions,
   } = useAutocomplete({ ...props, componentName: 'Autocomplete' });
 
+  const muiFormControl = useFormControl();
+
+  let disabled = disabledProp;
+
+  if (typeof disabled === 'undefined') {
```

Comment:
We can use the `formControlState` helper instead of this. See for example how it is used in the [InputBase](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/InputBase/InputBase.js#L321) component. We should also take into account the other properties that the `useFormControl` provides, like `fullWidth` and `size` and include them as well in the `formControlState` helper as a third argument alongside with the `disabled` prop.

Overall, the implementation in the `InputBase` should be a good reference.

Reply:
Yes I agree with the helper.

But i'm a bit lost about where i have to use the `fcs` values.
What about `ownerState`? 
In `renderInput` ? `InputProps` ? `inputProps`?

Maybe we have to pass `error` and `required` props to ?